src/components/chatsDetailPage/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { PlusIcon, VoiceIcon, EmojiIcon, SendIcon } from '../../assets';
+import date from '../../utils/date';
+import { ChatType } from '../../types/types';
+import { useSetRecoilState } from 'recoil';
+import { chatsState } from '../../recoil/atom';
+
+export default function Footer() {
+  const [input, setInput] = useState('');
+  const setChats = useSetRecoilState(chatsState);
+
+  const currentId = parseInt(localStorage.getItem('userId') || '');
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    if (input.trim() !== '') {
+      storeChat(input);
+      setInput('');
+    }
+  }
+
+  function handleInput(event: React.ChangeEvent<HTMLInputElement>) {
+    setInput(event.target.value);
+  }
+
+  function storeChat(input: string) {
+    const newChat: ChatType = {
+      id: Date.now().toString(),
+      from: 0,
+      to: currentId,
+      details: { text: input, time: date()},
+    };
+    setChats((prevChats) => [...prevChats, newChat]);
+    const existingChats = JSON.parse(localStorage.getItem('bckup') || '[]');
+    localStorage.setItem('bckup', JSON.stringify([...existingChats, newChat]));
```

Comment:
react Hooks를 이용하면 LocalStorage에 접근하는 횟수를 줄이는데 도움이 될 것 같아요.

```suggestion
useEffect(() => {
  localStorage.setItem('bckup', JSON.stringify(chats));
}, [chats]);
```